Library/Homebrew/cmd/update.rb

```diff
@@ -36,13 +45,27 @@ def update
         rescue
           onoe "Failed to update tap: #{tap}"
         else
-          report.update(updater.report) do |key, oldval, newval|
-            oldval.concat(newval)
+          updater.report
```

Comment:
Report inside thread is required as we want to speed up formula version checking.